docs/arch/adr-15.md

```diff
@@ -0,0 +1,40 @@
+# ADR 15: Drawing tools API
+
+June 19, 2019
+
+## Context
+
+The way the drawing tools currently function on Panoptes-Front-End (PFE) have numerous issues including:
+
+- Re-rendering the DOM more than necessary
+- Updating the classification annotation on each touch or click event
+
+Both of these can cause significant performance issues.
+
+- The separation concerns are not clear between components and stores. Multiple components can update what is being rendered making it hard to debug or add new features to. 
```

Comment:
Drawing is very difficult to debug but I'm not sure what this means, or which components it's referring to.

Reply:
I'll clarify this with an example.